docusaurus.config.js

```diff
@@ -175,7 +175,11 @@ const config = {
             ],
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
+        logo: {
+          alt: 'Fastify Logo',
+          src: 'img/logos/fastify-white.png',
```

Comment:
A bit huge:

<img width="1069" alt="image" src="https://user-images.githubusercontent.com/11404065/217734820-0f1a90a7-9392-4140-a674-e777f83de69f.png">

I think we need a smaller one or just remove it

Reply:
unfortunately cannot resize since the its just an `image-path` expected. 
i would be more intrigued to removing it and do away with the logo in footer since resizing constrained by docusaurus